stor/swift.py

```diff
@@ -253,8 +253,11 @@ def _swiftclient_error_to_descriptive_exception(exc):
     if exc_headers and exc_headers.get('X-Trans-Id'):
         exc_str += ' X-Trans-Id: %s' % exc_headers['X-Trans-Id']
     if http_status == 403:
-        logger.error('unauthorized error in swift operation - %s', exc_str)
-        return UnauthorizedError(exc_str, exc)
+        if "storage class" in exc_str:
```

Comment:
can you add a link to a doc that explains this

Reply:
added a small comment, otherwise I suggest looking at the test case